Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabStrip.css

```diff
@@ -1,192 +1,70 @@
 /* Style applied to the TabStrip. */
 
-/* Applied to the tab strip itself.*/
-*.tabStrip
-{
-}
-
-*.tabStripTabsPane
-{
-}
-
-*.tabStripTabsPane.empty
-{
-}
-
-*.tabStripTabsPane ul
+.tabStripTabsPane ul
 {
+  padding: var(--remotion-themed-spacing);
```

Comment:
why not padding:0?
(didn't live-test this right now.)

Reply:
`padding: var(--remotion-themed-spacing);` adds the gap between the left menu and the tabstrip

![image](https://user-images.githubusercontent.com/48289242/131810876-ee62e185-b219-4331-850c-f62612b16270.png)

If `padding: 0` the spacing has to be set in the parent eIement and not in the tabstrip itself.